beginning.md

```diff
@@ -310,18 +313,18 @@ if a:
 ```
 
 ## Functions
-* When we have to do a certain set of operatin multiple times and reuse it
-  with in codebase then we need to define a function
-* *def* keyword is used to define a function
-* syntex
+* When we have to do a certain set of operation multiple times and reuse it
+  with in codebase then we need to define a function.
+* `def` keyword is used to define a function.
+* Syntax:
 ```
 def function_name(function_arguments):
     do some stuff
     return something
 ```
 * Here function_arguments are not necessary.
 * function arguments can be passed in any fashion
-* if return is not define, it will not return anything.
+* If return is not define, it will not return anything.
```

Comment:
If any thing is missing, it will give syntax error.

Reply:
its all about return so yup you can say if return not define then return `None`